x-pack/plugin/sql/qa/src/main/resources/nested.csv-spec

```diff
@@ -106,23 +106,23 @@ SELECT first_name f, last_name l, YEAR(dep.from_date) start FROM test_emp WHERE
 
 f:s                  | l:s           | start:i
 
-Sreekrishna          |Servieres      |1985           
-Zhongwei             |Rosen          |1986           
-Chirstian            |Koblick        |1986           
-null                 |Chappelet      |1988           
-Zvonko               |Nyanchama      |1989     
+Parto                |Bamford        |1995
```

Comment:
Ordering here is completely wrong. 1995, 1986, 1996.... The query has `ORDER BY start` where `start` is `YEAR(dep.from_date)`.

Reply:
Oooops! I rushed here, sorry, need to check it. Thx for catching!